src/content/docs/apm/agents/python-agent/hosting-services/python-azure-touchless-integration.mdx

```diff
@@ -0,0 +1,308 @@
+---
```

Comment:
Where would you like this doc to appear in the left nav? @lrafeei

Reply:
Good question.  I was thinking the hosting section.  Perhaps right below the "direct installation" Azure doc in the hosting section?